meta-balena-common/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -8,7 +8,7 @@ SRC_URI:append = " \
     file://vacuum.conf \
     file://watchdog.conf \
     file://os.conf \
-    file://60-resin-update-state.rules \
+    file://65-resin-update-state.rules \
```

Comment:
Why is this re-ordering needed?

Reply:
@alexgg By default, the ordering of resin-update-state runs `resin_update_state_probe` before md arrays are assembled by `64-md-raid-assembly.rules`, which results in the by-state links not being created.

Reply:
thanks @jakogut should have read the commit message